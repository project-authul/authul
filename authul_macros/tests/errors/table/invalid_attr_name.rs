#[authul_macros::authul_table(blargh)]
struct Foo;

fn main() {
}
