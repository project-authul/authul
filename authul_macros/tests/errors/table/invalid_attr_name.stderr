error: invalid macro arg; expected 'name'
 --> tests/errors/table/invalid_attr_name.rs:1:31
  |
1 | #[authul_macros::authul_table(blargh)]
  |                               ^^^^^^
